scripts/run_lighthouse_tests.py

```diff
@@ -105,7 +105,10 @@ def main():
     atexit.register(cleanup)
 
     python_utils.PRINT('Building files in production mode.')
-    build.main(args=['--prod_env'])
+    # We are using --source_maps here, so that we have atlease one ci check
+    # that builds using source maps in prod env. This is to ensure that
+    # there are no issues while deploying oppia.
```

Comment:
```suggestion
    # We are using --source_maps here, so that we have at least one CI check
    # that builds using source maps in prod env. This is to ensure that
    # there are no issues while deploying oppia.
```

Reply:
done